docs/plugins/builtin/content-providers/blockquote.md

```diff
@@ -27,8 +27,9 @@ The content provider doesn't support any configuration parameters.
 The content provider supports the following execution parameters:
 
 ```hcl
-content blockquote {
-    value = <string>  # required
+content "blockquote" {
```

Comment:
one note--  could we drop the quotes around the data sink / content provider name in the docs? At the moment we use quotes for names mostly and it creates a nice implicit separation between Fabric's and user's names